src/api.rs

```diff
@@ -68,21 +68,30 @@ pub fn initialize_sta() -> Result<(), windows::core::Error> {
 }
 
 /// Audio direction, playback or capture.
-#[derive(Clone)]
+#[derive(Clone, Debug, PartialEq)]
```

Comment:
Is clippy happy with this, or does it suggest to also derive Eq?

Reply:
cargo clippy does not complain